core/templates/dev/head/components/forms/schema_editors/SchemaBasedFloatEditorDirective.js

```diff
@@ -32,13 +32,14 @@ oppia.directive('schemaBasedFloatEditor', [
         'schema_based_float_editor_directive.html'),
       restrict: 'E',
       controller: [
-        '$scope', '$filter', '$timeout', 'focusService',
-        function($scope, $filter, $timeout, focusService) {
+        '$scope', '$filter', '$timeout', 'FocusManagerService',
+        function($scope, $filter, $timeout, FocusManagerService) {
           $scope.hasLoaded = false;
           $scope.isUserCurrentlyTyping = false;
           $scope.hasFocusedAtLeastOnce = false;
 
-          $scope.labelForErrorFocusTarget = focusService.generateFocusLabel();
+          $scope.labelForErrorFocusTarget =
+          FocusManagerService.generateFocusLabel();
```

Comment:
Please indent it once from $scope, like

![indent](https://user-images.githubusercontent.com/7807557/31307197-5d025062-ab7c-11e7-97cc-fbd8faeb1f2c.jpeg)